app/assets/javascripts/components/common/date_picker.jsx

```diff
@@ -9,10 +9,11 @@ const DatePicker = React.createClass({
 
   propTypes: {
     id: React.PropTypes.string,
-    value: React.PropTypes.string,
+    value: React.PropTypes.object, // should be a moment object
```

Comment:
I didn't have luck with that, e.g. `moment().constructor.name === "n"` whatever the heck "n" is, not sure!